tuxemon/core/states/combat/combat.py

```diff
@@ -558,20 +560,31 @@ def perform_action(self, user, technique, target=None):
                 self._damage_map[target].add(user)
 
             else:  # assume this was an item used
-                if result:
-                    message += "\n" + trans('item_success')
+                if result["name"] == "capture":
```

Comment:
Traceback (most recent call last):
  File "C:/Users/Leif/PyCharmProjects/Tuxemon/tuxemon/main.py", line 38, in <module>
    main()
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\main.py", line 109, in main
    control.main()
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\control.py", line 519, in main
    self.main_loop()
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\control.py", line 580, in main_loop
    self.update(time_delta)
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\control.py", line 203, in update
    state.update(dt)
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\states\combat\combat.py", line 155, in update
    self.update_phase()
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\states\combat\combat.py", line 332, in update_phase
    self.handle_action_queue()
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\states\combat\combat.py", line 344, in handle_action_queue
    self.perform_action(*action)
  File "C:\Users\Leif\PyCharmProjects\Tuxemon\tuxemon\core\states\combat\combat.py", line 563, in perform_action
    if result["name"] == "capture":
KeyError: 'name'